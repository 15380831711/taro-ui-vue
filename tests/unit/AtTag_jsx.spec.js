import { shallowMount } from '@vue/test-utils'
import AtTag from '../demo/AtTag.jsx'

describe('AtTag.jsx', () => {
  it('snapshot test', () => {
    const wrapper = shallowMount(AtTag)
    expect(wrapper.element).toMatchSnapshot()
  })
})
