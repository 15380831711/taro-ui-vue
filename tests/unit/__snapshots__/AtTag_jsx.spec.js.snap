// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtTag.jsx snapshot test 1`] = `
<view>
  <text>
    AtTag
  </text>
</view>
`;
